genie/input_to_database.py

```diff
@@ -602,27 +607,26 @@ def validation(syn, center, process,
         add_query_str = "and name not like '%.vcf'" if process != "vcf" else ''
 
         validation_status_table = syn.tableQuery(
-            "SELECT id,md5,status,name,center,modifiedOn FROM {synid} "
+            "SELECT id,md5,status,name,center,modifiedOn,fileType FROM {synid} "
             "where center = '{center}' {add}".format(
                 synid=validation_status_synid,
                 center=center,
                 add=add_query_str))
-        validation_statusdf = validation_status_table.asDataFrame()
+
         error_tracker_table = syn.tableQuery(
             "SELECT id,center,errors,name FROM {synid} "
             "where center = '{center}' {add}".format(
                 synid=error_tracker_synid,
                 center=center,
                 add=add_query_str))
-        error_trackerdf = error_tracker_table.asDataFrame()
 
         input_valid_statuses = []
         invalid_errors = []
 
         for ents in center_files:
             status, errors = validatefile(syn, ents,
-                                          validation_statusdf,
-                                          error_trackerdf,
+                                          validation_status_table,
```

Comment:
Need to edit the doc string in `validatefile`. Currently it says it takes in the dataframe.